File: nodejs/src/hello.js
console.log('Hello, world!')

const helloWorld = 'Hello, world!'

if (helloWorld.includes('Hello')) {
  console.log(helloWorld)
}

const worldList = ['a', 'b', 'c']

for (const world of worldList) {
  if (world === 'a') {
    console.log('world is a!')
  } else if (world === 'b') {
    console.log('world is b!')
  } else {
    console.log('world is c!')
  }
}

const worldObject = {
  a: 'aaa',
  b: 'bbb',
  c: 'ccc'
}

for (const [key, value] of Object.entries(worldObject)) {
  console.log(`world ${key} is ${value}!`)
}

File: python/src/hello.py
"""
Say hello.
"""

__author__ = "RyoWakabayashi <gfdmkm573@gmail.com>"
__version__ = "1.0.0"
__date__ = "2021/04/20"

print("Hello, world!")

HELLO_WORLD = "Hello, world!"

if "Hello" in HELLO_WORLD:
    print(HELLO_WORLD)

world_list = ["a", "b", "c"]

for world in world_list:
    if world == "a":
        print("world is a!")
    elif world == "b":
        print("world is b!")
    else:
        print("world is c!")

world_dict = {"a": "aaa", "b": "bbb", "c": "ccc"}

for key, value in world_dict.items():
    print(f"world {key} is {value}!")
